README.md

```diff
@@ -15,19 +15,34 @@ Please note that LX Studio is not open-source software. The license grants permi
 
 ### Getting Started ###
 
-Rainbow Studio runs on top of Processing. [Download and install Processing &rarr;](https://processing.org/download/)
-
-Next, clone this project and open the RainbowStudio.pde project in Processing:
-```
-$ cd ~/Documents/Processing
-$ git clone https://github.com/tracyscott/RainbowStudio.git
-```
-
-The Animated GIF Pattern requires the [GifAnimation](https://github.com/01010101/GifAnimation) Processing Library. You need to install it into your Processing libraries folder. You might need to mkdir ~/Documents/Processing/libraries.
-```
-$ cd ~/Documents/Processing/libraries
-$ git clone https://github.com/01010101/GifAnimation.git
-```
+To set up RainbowStudio, perform the following steps:
+
+- [Download and install Processing](https://processing.org/download/)
+    - Rainbow Studio runs on top of Processing
+- Open Processing
+- Open its "Preference" and find out the "Sketchbook location"
+    - Example: `/home/todd/sketchbook`
+- [Download and install GifAnimation](https://github.com/01010101/GifAnimation) to `libraries/` in Sketchbook location
+    - Example: `cd /home/todd/sketchbook/libraries && git clone https://github.com/01010101/GifAnimation.git`
+- Clone this project and open the `RainbowStudio.pde` project in Processing:
+    - Example:
+    - ```bash
+      # Clone our repo
+      cd ~/github
```

Comment:
Change cd ~/github to cd /home/todd/sketchbook for the example documentation since end-user might not have ~/github.

Reply:
👍 Will update. Prob going to switch to `~` for all `/home/todd` references in fact